lisa/tools/fdisk.py

```diff
@@ -27,6 +27,8 @@ def make_partition(
         disk_name: str,
         file_system: FileSystem = FileSystem.ext4,
         format: bool = True,
+        first_sector: str = "",
```

Comment:
It looks they are not used. The two number is hard to use. When is it needed? If it's to create two partitions, handle it in this method internally should be easier.

Reply:
some scenario used default size without specifying the partition size, in this case, need specify the size as 100G

Reply:
If it needs 100GB, the argument can be like `disk_size`, instead of the sector. The sector can be detected in this method.

Reply:
@squirrelsc update, please review, I will merge it in next release. Thanks.